### PR TITLE
Update Payment Facilities

### PR DESCRIPTION
**Pull request by ialorro**
Deleted Transferwise from list of "Facilities not accepted at this time" @TQuock

### CODE REVIEW

**Review by TQuock — CHANGES_REQUESTED** [review]
Are we also going to add additional language as to who qualifies for Tranferswise?

**Review by justinegeffen — DISMISSED** [review]
Looks good to me, thank you!

**Comment by jasonblais** [comment]
@ialorro heads up on the question from Tim above

**Comment by ialorro** [comment]
Thank you @jasonblais ! This was resolved in pull request #136.

**Comment by jasonblais** [comment]
Great! @TQuock if you can help approve if the changes look good, given #136

**Comment by justinegeffen** [comment]
Following up on this, @TQuock, are you happy for this to be merged? :)

File: company/how-to-guides-for-staff/how-to-purchase/how-to-on-board-as-a-vendor/how-to-get-paid.md [hunk 1]
```diff
@@ -18,23 +18,23 @@ If you are a vendor, refer to the [Bank Form](https://handbook.mattermost.com/co
...
+#### Rest of world
 
 If you are a vendor, or a contractor located outside the United States and Canada, Mattermost requires a primary banking institution as a method of payment, as we use Bill.com for invoice processing and payment. If you are a staff member located outside of the United States, Canada, United Kingdom and Germany, you have the option of getting paid either by bill.com (via wire transfer) or TransferWise for invoice processing, expense reimbursement and payment. The currency in which payment is made is based on the currency stated within the agreement signed with Mattermost.
 
 Primary Banking Institution and Bill.com
 
 The person or legal entity the bank account is registered to must be in the name of the contracting individual/entity.
 
-_Example 1_: a staff member named John Smith is located in Spain and signed a consulting agreement with Mattermost, Inc. under his surname, John Smith must have the following banking institution setup:
+_Example 1_: A staff member named John Smith is located in Spain and signed a consulting agreement with Mattermost, Inc. under his surname, John Smith must have the following banking institution setup:
```
Comment (TQuock) [inline company/how-to-guides-for-staff/how-to-purchase/how-to-on-board-as-a-vendor/how-to-get-paid.md:29]:
```suggestion
_Example 1_: A staff member named John Smith is located in Spain and signed a consulting agreement with Mattermost, Inc. under his surname, John Smith will need the following banking institution setup:
```

File: company/how-to-guides-for-staff/how-to-purchase/how-to-on-board-as-a-vendor/how-to-get-paid.md [hunk 2]
```diff
@@ -18,23 +18,23 @@ If you are a vendor, refer to the [Bank Form](https://handbook.mattermost.com/co
...
 
-* The bank account is located in Spain
-* The bank account is his name or other known surname
+* The bank account is located in Spain.
+* The bank account is his name or other known surname.
 
-_Example 2_: a staff member named John Smith is located in Spain and signed a consulting agreement with Mattermost, Inc. under his LLC titled Better Coding, John Smith must have the following banking institution setup:
+_Example 2_: A staff member named John Smith is located in Spain and signed a consulting agreement with Mattermost, Inc. under his LLC titled Better Coding, John Smith must have the following banking institution setup:
```
Comment (TQuock) [inline company/how-to-guides-for-staff/how-to-purchase/how-to-on-board-as-a-vendor/how-to-get-paid.md:34]:
```suggestion
_Example 2_: A staff member named John Smith is located in Spain and signed a consulting agreement with Mattermost, Inc. under his LLC titled Better Coding, John Smith will need the following banking institution setup:
```

File: company/how-to-guides-for-staff/how-to-purchase/how-to-on-board-as-a-vendor/how-to-get-paid.md [hunk 3]
```diff
@@ -18,23 +18,23 @@ If you are a vendor, refer to the [Bank Form](https://handbook.mattermost.com/co
...
 If you are a staff member located in Canada or if your contract is with Open World Services \(OWS\), payroll will be made to you in accordance with OWS policy. For expense reimbursement, please follow the [OWS Reimbursement Process](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-spend-company-money/how-to-use-expensify#open-world-services-ows).
 
-#### Rest of World
+#### Rest of world
 
 If you are a vendor, or a contractor located outside the United States and Canada, Mattermost requires a primary banking institution as a method of payment, as we use Bill.com for invoice processing and payment. If you are a staff member located outside of the United States, Canada, United Kingdom and Germany, you have the option of getting paid either by bill.com (via wire transfer) or TransferWise for invoice processing, expense reimbursement and payment. The currency in which payment is made is based on the currency stated within the agreement signed with Mattermost.
 
 Primary Banking Institution and Bill.com
 
 The person or legal entity the bank account is registered to must be in the name of the contracting individual/entity.
```
Comment (TQuock) [inline company/how-to-guides-for-staff/how-to-purchase/how-to-on-board-as-a-vendor/how-to-get-paid.md:27]:
```suggestion
The person or legal entity the bank account is registered to will need to be in the name of the contracting individual/entity.
```

File: company/how-to-guides-for-staff/how-to-purchase/how-to-on-board-as-a-vendor/how-to-get-paid.md [hunk 4]
```diff
@@ -18,23 +18,23 @@ If you are a vendor, refer to the [Bank Form](https://handbook.mattermost.com/co
...
 
-_Example 2_: a staff member named John Smith is located in Spain and signed a consulting agreement with Mattermost, Inc. under his LLC titled Better Coding, John Smith must have the following banking institution setup:
+_Example 2_: A staff member named John Smith is located in Spain and signed a consulting agreement with Mattermost, Inc. under his LLC titled Better Coding, John Smith must have the following banking institution setup:
 
-* The bank account is located in Spain
-* The bank account is in the name of Better Coding, LLC
+* The bank account is located in Spain.
+* The bank account is in the name of Better Coding, LLC.
 
 If you provide information for your banking institution, your invoice and/or expense reimbursements will be made through Bill.com. As noted above, Mattermost requires banking information from a primary banking institution to be able to make reimbursements through Bill.com. Approved expense reports are reimbursed either by the 15th or end of each month.
```
Comment (TQuock) [inline company/how-to-guides-for-staff/how-to-purchase/how-to-on-board-as-a-vendor/how-to-get-paid.md:39]:
```suggestion
If you provide information for your banking institution, your invoice and/or expense reimbursements will be made through Bill.com. As noted above, Mattermost needs banking information from a primary banking institution to be able to make reimbursements through Bill.com. Approved expense reports will follow our [Expense reimbursement policy](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-spend-company-money/how-to-use-expensify#expense-reimbursement-policy).
```

File: company/how-to-guides-for-staff/how-to-purchase/how-to-on-board-as-a-vendor/how-to-get-paid.md (hunk 3 above)
Comment (TQuock) [reply company/how-to-guides-for-staff/how-to-purchase/how-to-on-board-as-a-vendor/how-to-get-paid.md:27]: @ialorro - can you please check my suggestions and make sure that you agree? If you are good, I'll commit these

**Comment by ialorro** [comment]
duplicate pull request - resolved in #136

**Comment by TQuock** [comment]
@ialorro did all the updates make it to the other commit?

**Comment by ialorro** [comment]
@TQuock I believe so, this was the first pull request I created (not really knowing how to use GitHub) and then I created #136, which had all the edits needed for this section.

**Comment by TQuock** [comment]
Thanks for the clarification. Issue is closed on my end

All the best,

*Tim Quock *
Director of Accounting
tim@mattermost.com | LinkedIn
<https://www.linkedin.com/in/timothy-quock-23300628/> | mattermost.com | google
calendar availability <https://calendly.com/matters/30min-1>

O: (650) 667-8512
C: (925) 878-1338



*"Train people well enough so they can leave, treat people well enough so
they don't want to" - Richard Branson *

**Pull request closed without merging**